lib/edusign/client.rb

```diff
@@ -1,5 +1,278 @@
 module Edusign
   class Client
-    # TODO(grmnlrt): put Kitt code here.
+    include HTTParty
```

Comment:
Needs to be declarend as dependency in the gemspec

Reply:
I think so!

Reply:
Also maybe

```
bundle lock --add-platform x86_64-linux
```

cf GHA logs

Reply:
Actually I needed to `require "httparty"` as well 😅 
I forgot I wasn't on rails